typing/btype.mli

```diff
@@ -227,6 +227,8 @@ val undo_compress: snapshot -> unit
 val link_type: type_expr -> type_expr -> unit
         (* Set the desc field of [t1] to [Tlink t2], logging the old
            value if there is an active snapshot *)
+val set_type:  type_expr -> type_desc -> unit
```

Comment:
I would name this `set_desc`.

Reply:
This was my original intention. I changed that for two reasons:
* set_desc is not very descriptive
* I had written set_desc in the interface, but then changed all the occurrences to set_type
If you really insist I can call it set_type_desc.

Reply:
`set_type_desc` seems fine indeed :)

Sorry for the trouble.

Reply:
Done